peopleops/index.md

```diff
@@ -115,3 +115,15 @@ The company covering certain costs, such as broadband can have tax implications
 If you want to occasionally use a separate workspace to your home, the company will cover up to $150 a month in charges for renting a desk in a co-working space. 
 
 [gh-admin]: https://github.com/flowforge/admin
+
+
+#### Mileage
+
+Use of a personal vehicle for business purposes, other than commuting to a permanent workplace, is covered according to the employees local laws.
+
+For example:
```

Comment:
Both are a maximum allowed, am I right?

Can we just standardise on a dollar / km price? Our accounting is in dollars, and most countries around the world use kms.

Actually, @Steve-Mcl just topped up his tank with his Brex card. Are you suggesting to no longer do that?

Reply:
Yes, the two examples I've added are based on the maximum allowable rate. Every company I've worked with in the UK uses this rate, I'm not sure about the Netherlands, what's your experience there? Or with GitLab? The rate is coupled with not causing a taxable event for any reimbursement, which is why it's usually based on where the employee is legally employed. I'm not sure how that would work if FlowForge put together a global rate, though I suppose that's theoretically possible!

I'd suggest not putting fuel directly on the company card as it could potentially become taxable. Additionally, the allowable HMRC amount is designed to cover wear on the vehicle, insurance costs, road tax, maintenance and fuel; so usually works out slightly more than expensing just the fuel.

Reply:
I just submitted my expenses including petrol, I used the UK standard model of £.45 per mile up to 10,000 miles per year as set out here - https://www.gov.uk/tax-relief-for-employees/vehicles-you-use-for-work

I don't think we can do it in $/KM due to fluctuating exchange rates. If someone claims over £.45 I think the UK tax man could get unhappy.

I'd not recommend using a company card to pay for petrol, it's not considered the 'right' way to do it by HMRC so could cause issues.

Could we just say 'claims for mileage in personal cars should be made using the appropriate calculations based on the country you are taxed in'. We could then add the links to local government advice on a case by case basis (so start with UK, US, Belgium and the Netherlands) then add more as people join?

PS, 100% agree with everything @Pezmc said.

Reply:
I've always felt the 0.19 cents per KM in NL is excessive. For reference, our own vehicle costs about 7 euro per 100km to drive. Whatever the UK now picked as a number is for me weirdly high considering petrol is cheaper in the UK. Having the steering wheel on the wrong side I bet is more expensive to maintain, but not that expensive?

Another note; when I drove to the UK this spring I didn't expense my KMs, but if I did, I got less per km than my colleague from the UK? Makes very little sense. 

Anyway, I'll stop bikeshedding and will merge as is. But I do think this is a policy that will need standardisation some day to one figure per KM.